documentation/example-session.md

```diff
@@ -0,0 +1,43 @@
+```js
+const Arc = require('@daostack/client').Arc
+
+const arc = new Arc({
+  graphqlProvider: 'https://url.to.graphql.subgraph/',
+  web3Provider: 'https://url.to.web3.node/',
+})
+
+// get an Observable list of DAOs
+const daoObservable = arc.daos()
+// or just get the list itself
+const daos = await arc.daos().toPromise()
+
+// you can query for a particular DAO
+dsc.daos({ name: 'mydao'})
```

Comment:
I think `dsc` should be `arc` ? (also in the row below)

Reply:
> I think `dsc` should be `arc` ? (also in the row below)

yes